include/iso_alloc_ds.h

```diff
@@ -0,0 +1,107 @@
+/* iso_alloc_ds.h - A secure memory allocator
+ * Copyright 2022 - chris.rohlf@gmail.com */
+
+#pragma once
```

Comment:
It this standard and supported everywhere?

Reply:
Yes https://en.wikipedia.org/wiki/Pragma_once#Portability